### PR TITLE
Adding filter to get-many on instances

### PR DESCRIPTION
**Pull request by juliasaveliff**
For instances: 
- Separate sorting logic 
- Implement filtering (except for on rack range)

### CODE REVIEW

File: rackcity/views/it_instance_views.py [hunk 1]
```diff
@@ -205,3 +208,72 @@ def is_location_full(rack_id, instance_elevation, instance_height):
...
+                elif filter_dict['match_type'] == 'contains':
+                    filter_args['{0}__icontains'.format(filter_field)] = \
+                        filter_dict['value']
+
+            elif filter_type == 'numeric':
+                range_value = (
+                    int(filter_dict['min']),
+                    int(filter_dict['max'])
+                )
+                filter_args['{0}__range'.format(filter_field)] = range_value
```
Comment (benauriemma) [inline rackcity/views/it_instance_views.py]: Can you leave a comment about whether this is inclusive on min/max?

File: rackcity/views/it_instance_views.py [hunk 2]
```diff
@@ -205,3 +208,72 @@ def is_location_full(rack_id, instance_elevation, instance_height):
...
+
+            elif filter_type == 'rack_range':
```
Comment (benauriemma) [inline rackcity/views/it_instance_views.py]: Do we have a ticket to add this in later?

Comment (juliasaveliff) [reply rackcity/views/it_instance_views.py]: just made one!

File: rackcity/views/it_instance_views.py (hunk 1 above)
Comment (juliasaveliff) [reply rackcity/views/it_instance_views.py]: adding comment

**Pull request merged**